src/chapter_0/the_pillars.md

```diff
@@ -1 +1,154 @@
 # The pillars
+My dear reader, welcome to the cornerstone of your programming enlightenment--or whatever semblance of it you'll achieve after wading through this chapter. Here, we dissect the four almighty pillars that hold up the shaky (very shaky) roof of the world: **Mathematics**, **Computer Networks**, **Algorithms & Data Structures** and **Operating Systems**. And yes, don't you worry, we'll tell you where blockchain fits into all this
```

Comment:
Exists a continue error because, at any moment, blockchain was mentioned as a fundamental part of this book. I think we still need the “object_study.md” because I don't say a shit about this in the intro LOL